airbyte-integrations/connectors/source-gnews/setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1",
+    "airbyte-cdk~=0.13.2",
```

Comment:
We should remove the patch version from here. Right now, this will cause us to only pull the `airbyte-cdk` from `0.13.x` onward as per [PEP 440](https://peps.python.org/pep-0440/#compatible-release). Whereas ideally we probably want to continue to get minor bumps as the CDK continues to be changed in a non-breaking way.

Reply:
@brianjlai this change does rely on CDK version 0.13.2 or greater. WDYT about changing it to `airbyte-cdk>=0.13.2`?

Reply:
I agree that we should definitely pin it to use `0.13` or above since it relies on a new feature, but the problem with explicitly marking it with the patch version `2` is that next time we build, we will only pull `airbyte-cdk` where minor version is `13`. So in the next month if we progress to CDK `0.20.0`, this connector will continue to be pinned at `0.13.x` since it will only look for the latest patch. And I think based on PEP 440 `~=` and `>=` are interchangeable. Basically I think we want to continue to get latest minors and specifying a patch in setup.py will inhibit that.

And actually given that this is a new non-breaking feature, we should probably also be bumping this to `0.14.0`.

Reply:
Got it, thanks for clarifying. Sounds like I should have bumped the CDK change to `0.14.0`. Looks like that does exist now so I'll change this to use `airbyte-cdk~=0.14`.

Reply:
ah sorry my bad, I was getting some PRs mixed. Up this was more that in hindsight that we should've minor version bumped the cdk schema change to 14 since it was a feature add. But regardless `0.14.0` is still fine since that was released earlier today.